src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -33,12 +34,32 @@ public sealed partial class ColumnViewBase : BaseLayout
 
 		protected override ItemsControl ItemsControl => FileList;
 
+		private ColumnViewBrowser? columnsOwner;
+		private ListViewItem? openedFolderPresenter;
+
 		public ColumnViewBase() : base()
 		{
 			this.InitializeComponent();
 			var selectionRectangle = RectangleSelection.Create(FileList, SelectionRectangle, FileList_SelectionChanged);
 			selectionRectangle.SelectionEnded += SelectionRectangle_SelectionEnded;
 			tapDebounceTimer = DispatcherQueue.CreateTimer();
+			this.ItemInvoked += ColumnViewBase_ItemInvoked;
+		}
+
+		private void ColumnViewBase_ItemInvoked(object? sender, EventArgs e)
+		{
+			ClearOpenedFolderSelectionIndicator();
+			openedFolderPresenter = FileList.ContainerFromItem(FileList.SelectedItem) as ListViewItem;
+		}
+
+		private void ClearOpenedFolderSelectionIndicator()
+		{
+			if (openedFolderPresenter != null)
+			{
+				openedFolderPresenter.Background = new SolidColorBrush(Microsoft.UI.Colors.Transparent);
+				var presenter = openedFolderPresenter.FindDescendant<Grid>()!;
+				presenter!.Background = new SolidColorBrush(Microsoft.UI.Colors.Transparent);
+			}
```

Comment:
I'd invert the condition in the `if` to get a `if(openedFolderPresenter is null) return` instead.